# Patch is from RedHat binutils-2.25.1 src rpm.
# No need to send to binutils, they know about it.
--- a/bfd/srec.c	2014-11-04 01:54:41.000000000 -0800
+++ b/bfd/srec.c	2015-11-14 22:32:41.094053576 -0800
@@ -249,7 +249,7 @@
       char buf[40];
 
       if (! ISPRINT (c))
-	sprintf (buf, "\\%03o", (unsigned int) c);
+	sprintf (buf, "\\%03o", (unsigned int) (c & 0xff));
       else
 	{
 	  buf[0] = c;
